internal/server/handlers_accounts.go

```diff
@@ -165,11 +157,9 @@ func CreateAccount(env *HandlerConfig) http.HandlerFunc {
 			writeError(w, http.StatusInternalServerError, "Internal Error")
 			return
 		}
-		accountResponse := CreateAccountResponse{
-			ID: int(id),
-		}
+		successResponse := SuccessResponse{Message: "success"}
 		w.WriteHeader(http.StatusCreated)
-		err = writeJSON(w, accountResponse)
+		err = writeJSON(w, successResponse)
```

Comment:
Shouldn't Notary include the account ID in the Create Account response?

Reply:
that value is no longer returned by sqlair